scripts/install/autobrr.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+# autobrr installer
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+app_reqs=("curl" "sqlite3")
+apt_install "${app_reqs[@]}"
```

Comment:
I see where you pulled this from, but that's an "arr" format, tl;dr:
```suggestion
apt_install curl sqlite3
```